sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -29,6 +30,23 @@ export type AuthorizationRule = {
     secondaryKey?: string;
 };
 
+// Warning: (ae-forgotten-export) The symbol "SubscriptionRuleManagement" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type ClientTypeT<ReceiveModeT extends "peekLock" | "receiveAndDelete", EntityTypeT extends "queue" | "subscription", SessionsEnabledT extends "sessions" | "nosessions"> = SessionsEnabledT extends "nosessions" ? EntityTypeT extends "queue" ? NonSessionReceiverTrack2<ReceiveModeT> : NonSessionReceiverTrack2<ReceiveModeT> & SubscriptionRuleManagement : EntityTypeT extends "queue" ? SessionReceiverTrack2<ReceiveModeT> : SessionReceiverTrack2<ReceiveModeT> & SubscriptionRuleManagement;
+
+// @public
+export interface ContextWithSettlement {
+    // (undocumented)
+    abandon(m: Message): Promise<void>;
+    // (undocumented)
+    complete(m: Message): Promise<void>;
+    // (undocumented)
+    deadLetter(m: Message): Promise<void>;
+    // (undocumented)
```

Comment:
Kind reminder to document stuff. Perhaps on another PR! 🌞

Reply:
It's a criteria for preview 1 - I'll be pushing more docs for this PR.

Reply:
All exported types have been documented.